src/rules/noUnnecessaryTypeAssertionRule.ts

```diff
@@ -36,13 +42,13 @@ export class Rule extends Lint.Rules.TypedRule {
     public static FAILURE_STRING = "This assertion is unnecessary since it does not change the type of the expression.";
 
     public applyWithProgram(sourceFile: ts.SourceFile, program: ts.Program): Lint.RuleFailure[] {
-        return this.applyWithWalker(new Walker(sourceFile, this.ruleName, program.getTypeChecker()));
+        return this.applyWithWalker(new Walker(sourceFile, this.ruleName, this.getOptions(), program.getTypeChecker()));
```

Comment:
you can use `this.ruleArguments` instead of `this.getOptions()`
you just need to change the type argument from `Lint.IOptions` to `string[]`